implementation.md

```diff
@@ -1554,3 +1554,268 @@ In the `Video` and `Audio` objects, sellers must only include one form of floor
 ### 7.11.2 - How Buyers Should Interpret Floor Guidance
 
 Buyers bidding with a specific Deal ID should use the floor guidance provided in the corresponding `Deal` object. If there is no floor guidance in the `Deal` object or the buyer is bidding on an Open Market impression opportunity, the buyer should use the floor guidance provided in the corresponding `Video` or `Audio` object to inform their bids. Finally, if no floors are provided in the `Video` or `Audio` objects or the buyer is bidding on a Native or Banner impression opportunity, the buyer should use the floor guidance provided in the `Imp` object.
+
+## 7.12 - ID Match Method Guidance <a name="idmm"></a>
+
+### 7.12.1 - Best Practices for Disclosing ID Matches
+
+Unless prior arrangements have been made between the buyer and the seller directly, the value in the <code>user.buyeruid</code> field is expected to be derived from a real time cookie sync (see Appendix: Cookie Syncing) and value in <code>device.ifa</code> field is expected to be derived from an advertising ID call to the Operating System. 
+
+For an Exchange to properly disclose that an ID substitution has occurred by a publisher, they must propagate the <code>mm</code> value, as sent by the publisher, on the Extended Identifier (eid) they send to the DSP, regardless of how the SSP determined the DSP’s id.
```

Comment:
I think you can get rid of this 1-sentence paragraph, because the following paragraph describes this exact same scenario.

Reply:
i think the more times we repeat it the better